Sources/Webservice/HTTP/Headers/ContentType.swift

```diff
@@ -34,6 +34,9 @@ public struct JSONContentType: ContentType {
     }
 
     private let charSet: CharSet?
+    private let _encoder: JSONEncoder
```

Comment:
Should not `private (set) var encoder` got you the same access level instead of defining _encoder?

Reply:
Actually it needs to be:

` public private (set) var encoder: ContentTypeEncoder?`
` public private (set) var decoder: ContentTypeDecoder?`

I'll update so it's cleaner.